vimeo-networking/src/main/java/com/vimeo/networking/VimeoClient.java

```diff
@@ -871,8 +907,9 @@ public void run() {
                     VimeoClient.sContinuePinCodeAuthorizationRefreshCycle = false;
                     mTimer.cancel();
                     if (authCallback != null && now >= mExpiresInNano) {
-                        final VimeoError error = new VimeoError("Pin code expired.");
-                        error.setLocalErrorCode(LocalErrorCode.UNABLE_TO_LOGIN_PINCODE_EXPIRED);
+                        final VimeoResponse.Error error = new VimeoResponse.Error.Exception(
```

Comment:
It certainly does break the current pincode code, but I provided a very simple alternative approach. Instead of the old `vimeoError.localErrorCode == LocalErrorCode.UNABLE_TO_LOGIN_PINCODE_EXPIRED` check, the new code would just check `vimeoError is VimeoResponse.Error.Exception && vimeoEror.throwable is PinCodeExpiredException`. It's a direct conversion that's easy to do. I'm going to leave it in the code base for now.